src/services/ip-address/ipAddress.service.ts

```diff
@@ -0,0 +1,18 @@
+import 'reflect-metadata'
+
+import { Request } from 'express'
+import { provide } from 'inversify-binding-decorators'
+
+@provide(IpAddressService)
+export default class IpAddressService {
+
+  public getIpAddress(req: Request): string {
+    const forwardedIpsStr = req.headers['x-forwarded-for'] as string
+    let ip = ''
+
+    if (forwardedIpsStr) {
+        ip = forwardedIpsStr.split(',')[0]
+    }
+    return ip
+  }
```

Comment:
add type to the `forwardedIpsStr` variable please. Your code would indicate that it is `Optional<string>`

Also, I think you can change this to:
```
const ip: string = forwardedIpsStr?.split?[0] || ''
```

Can you confirm a use case where the x-forwarded-for header is not present? If not, please confirm with someone from the platform team maybe i.e. @klawani . If there is no use case, consider removing the nullable check and assuming that it is mandatory.